docs/GettingStartedTensorflow.md

```diff
@@ -0,0 +1,244 @@
+
+### Tensors
+
+The network are fed with TensorFlow Tensor objects that are in [N, H, W, C] format.
+
+## Other TensorFlow Tools in DIGITS
+
+DIGITS provides a few useful tools to help with your development with TensorFlow.
+
+### Provided Helpful Functions
+
+DIGITS provides a few helpful functions to help you with creating the model. Here are the functions we provide inside the `digits` class
+
+Function Name       | Parameters          | Description
+--------------------|---------------------|-------------
+`classification_loss` | pred - the images to be classified <br> y - the labels | Used for classification training to calculate the loss of image classification
+`mse_loss`            | lhs - left hand tensor <br> rhs - right hand tensor | Used for calculating the mean square loss between 2 tensors
+`constrastive_loss`   | lhs - left hand tensor <br> rhs - right hand tensor <br> y - `labels` | Calculates the contrastive loss with respect to the Caffe definition
+`classification_accuracy` | pred - the image to be classified <br> y - the labels | Used to measure how accurate the classification task is
+`nhwc_to_nchw` | x - the tensor to transpose | Transpose the tensor that was originally NHWC format to NCHW. The tensor must be a degree of 4
+`nchw_to_nhwc` | x - the tensor to transpose | Transpose the tensor that was originally NCHW format to NHWC. The tensor must be a degree of 4
+`hwc_to_chw` | x - the tensor to transpose | Transpose the tensor that was originally HWC format to CHW. The tensor must be a degree of 3
+`chw_to_hwc` | x - the tensor to transpose | Transpose the tensor that was originally CHW format to HWC. The tensor must be a degree of 3
+`bgr_to_rgb` | x - the tensor to transform | Transform the tensor that was originally in BGR channels to RGB.
+`rgb_to_bgr` | x - the tensor to transform | Transform the tensor that was originally in RGB channels to BGR.
+
+### Visualization With TensorBoard
+
+![TensorBoard](images/TensorBoard.png)
+
+TensorBoard is a visualization tools provided by TensorFlow to see the graph of your neural network. DIGITS provides easy access to TensorBoard network visualization for your network while creating it. This can be accessed by clicking on the ```Visualize``` button under ```Custom Network``` as seen in the image below.
+
+![Visualize TensorBoard](images/visualize_button.png)
+
+If there is something wrong with the network model, DIGITS will automatically provide with you the stacktrace and the error message to help you understand where the problem is.
+
+You can also spin up the full Tensorboard server while your model is training with the command
+```
+$ tensorboard --logdir <job_dir>/tb/
+```
+where `<job_dir>` is the directory where them model is being trained at, which can be found here:
+
+![Job Dir](images/job-dir.png)
+
+Afterwards, you can open up the Tensorboard page by going to
+```http://localhost:6006```
+
+Or you can click the ```Tensorboard``` link under Visualization
```

Comment:
Just a single backtick will do here. Triple backticks is just for multi-line code blocks.